tools/online_editor/preview.html

```diff
@@ -1,33 +1,43 @@
 <!DOCTYPE html>
 <html>
-
-<head>
-    <meta charset="utf-8">
+  <head>
```

Comment:
In curious, is there a specific reason these field and the scripts are indented with two spaces instead of the four we use for rust, C++ and cmake?

Reply:
No reason... its what falls out of the prettier, the first prettifier I could find for JS + co. It supposedly is rather popular...

I am open to use another one instead, I just do not want to bother fixing indentation all over the place manually.

Reply:
I’m not suggesting to use a different formatter, I think it would suffice to configure the formatter to use 4 instead of 2 for consistency with the rest of the source code in this repository. Or did I misunderstand and prettier cannot be configured to use 4?

Reply:
FWIW, if you prefer not to do this yourself, then I volunteer to submit a `.prettier` configuration that sets the indentation (tab) width for 4 and I'll reformat all the files. Just let me know if you'd like to do it or if you shall propose a PR for discussion.

Reply:
I can do that in master and then rebase on top of that:-)

Reply:
Great, thanks!

Reply:
@simon: See #1511